geomesa-convert/geomesa-convert-common/src/main/scala/org/locationtech/geomesa/convert/SimpleFeatureConverterFactory.scala

```diff
@@ -168,7 +182,9 @@ abstract class AbstractSimpleFeatureConverterFactory[I] extends SimpleFeatureCon
       } else {
         Seq("has-geo", "has-dtg")
       }
-    ValidatorLoader.createValidator(validators, sft)
+    val validator = SimpleFeatureValidator(validators)
```

Comment:
wondering if we should enable the z3 validator by default for converter2...

Reply:
or at least perhaps we look at the indexes and if z2 or z3 is present we enable z-index validator followon ticket if you think its a good idea?

Reply:
good idea. ticket created here: https://geomesa.atlassian.net/browse/GEOMESA-2318